test/XRP/xrp-protocol-buffer-conversion.test.ts

```diff
@@ -102,35 +105,61 @@ describe('Protocol Buffer Conversion', function (): void {
     // WHEN the protocol buffer is converted to a native TypeScript type.
-    const pathElement = XrpPathElement.from(testPathElementProto)
+    const pathElement = XrpPathElement.from(testPathElementProtoWithAccount)
 
     // THEN the currency converted as expected.
     assert.equal(
       pathElement.account,
-      testPathElementProto.getAccount()!.getAddress(),
+      testPathElementProtoWithAccount.getAccount()!.getAddress(),
     )
+    assert.equal(pathElement.currency, undefined)
+    assert.equal(pathElement.issuer, undefined)
+  })
+
+  it('Convert PathElement protobuf with two valid fields set to XrpPathElement', function (): void {
+    // GIVEN a PathElement protocol buffer with currency and issuer fields set.
+    // WHEN the protocol buffer is converted to a native TypeScript type.
+    const pathElement = XrpPathElement.from(
+      testPathElementProtoWithCurrencyIssuer,
+    )
+
+    // THEN the currency converted as expected.
+    assert.equal(pathElement.account, undefined)
     assert.deepEqual(
       pathElement.currency,
-      XrpCurrency.from(testPathElementProto.getCurrency()!),
+      XrpCurrency.from(testPathElementProtoWithCurrencyIssuer.getCurrency()!),
     )
     assert.equal(
       pathElement.issuer,
-      testPathElementProto.getIssuer()!.getAddress(),
+      testPathElementProtoWithCurrencyIssuer.getIssuer()!.getAddress(),
     )
   })
 
+  it('Convert PathElement protobuf with mutually exclusive fields set to XrpPathElement', function (): void {
+    // GIVEN a PathElement protocol buffer with account and currency fields set.
+    // WHEN the protocol buffer is converted to a native Typescript type THEN an error is thrown.
+    assert.throws(() => {
+      XrpPathElement.from(testInvalidPathElementWithAccountCurrency)
+    }, XrpError)
+  })
+
+  it('Convert PathElement protobuf with different mutually exclusive fields set to XrpPathElement', function (): void {
+    // GIVEN a PathElement protocol buffer with account and issuer fields set.
+    // WHEN the protocol buffer is converted to a native Typescript type THEN an error is thrown.
+    assert.throws(() => {
+      XrpPathElement.from(testInvalidPathElementWithAccountIssuer)
+    }, XrpError)
+  })
+
   it('Convert PathElement protobuf with no fields set to XrpPathElement', function (): void {
     // GIVEN a PathElement protocol buffer with no fields set.
-    // WHEN the protocol buffer is converted to a native TypeScript type.
-    const pathElement = XrpPathElement.from(testEmptyPathElementProto)
-
-    // THEN the currency converted as expected.
-    assert.isUndefined(pathElement.account)
-    assert.isUndefined(pathElement.currency)
-    assert.isUndefined(pathElement.issuer)
+    // WHEN the protocol buffer is converted to a native Typescript type THEN an error is thrown.
+    assert.throws(() => {
+      XrpPathElement.from(testInvalidPathElementProtoEmpty)
+    }, XrpError)
```

Comment:
yeah I think it's fine to omit the specific error string to make life easier.  maybe include it in a test if you have any reason to doubt that you've gotten your error messages right.